### PR TITLE
Accessibility checkout

### PR DESCRIPTION
**Pull request by josman227**
1. Page titles - Updated all pages to have descriptive and meaningful titles.
   
2. Image text alternatives - Added alt text to all images for better accessibility.
  
3. Text headings - Ensured proper heading hierarchy and used semantic headings.

4. Resize - Ensured the website remains usable and accessible at different screen sizes.
  
5. Moving content - Ensured content does not move unexpectedly, providing a stable user experience.
 
6. Multimedia - Provided captions and/or transcripts for multimedia content.
 
7. Basic structure of the page - Checked and ensured the correct use of HTML tags and semantic structure.

8. Edit a report file of accessibility.

9. Add labels for the contact form

I have thoroughly tested these changes and confirm that they improve the website's accessibility. Please review and merge this pull request.

### CODE REVIEW

**Review by Ofarouq310 — COMMENTED** [review]
Hi Team, ✋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

giphy

## 🔴 Invalid Code Review Request

- You have submitted a project, that should have been done by pair programming, which means that the three of you have to contribute equally to the project.

- Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html
Comment (Alusp) [inline index.html:1]: > `<form class="form" action="https://formspree.io/f/moqzgdak" method="post">
    <input type="text" name="name" id="name" maxlength="30" placeholder="Full Name">
    <input type="text" name="name" id="firstName" placeholder="First Name" maxlength="30">
    <input type="text" name="name" id="lastName" placeholder="Last Name" maxlength="30">
     <input name="email" id="email" type="email" placeholder="Email adress">
     <textarea  name="message" id="message" maxlength="500" cols="30" rows="5">Hey I’d like to connect to..</textarea>
     <a href="#"><button class="button form-btn" type="submit" >Get in touch </button></a>
        </form>`

- [ ] A very good job here so far :clap: However, Please add a label to each of the form input field as recommended in the project requirement. Kindly use this [link](https://www.w3schools.com/tags/tag_label.asp) for quick reference on how to implement it. :slightly_smiling_face:

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @Nessrine88, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 
close

## Highlights
- [ ] Good Commit History :green_circle: 
- [ ] Good Github Flow :green_circle: 
- [ ] Good PR Summary :green_circle: 
- [ ] No Linter Errors :green_circle: 


## Required Changes ♻️
_Check the comments under the review._
- [ ] A very good job so far :+1: However, Please use this [link](https://wave.webaim.org/) to resolve your contrast as its showing 17 contrast errors. To ensure a positive user experience, improve accessibility, and adhere to design best practices, it is important to pay attention to contrast ratios and ensure that text, icons, and other essential elements have sufficient contrast against their backgrounds. This typically involves choosing color combinations with significant contrast and conducting thorough testing to validate readability and accessibility. :slightly_smiling_face: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Nessrine88        👋

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Good job with the use of GitHub features✔️ 
- Kudos on implementing the contrast changes as requested ✔️


## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**